.github/workflows/conflicts.yml

```
@@ -0,0 +1,27 @@
+---
+name: Label Merge Conflicts
+
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - master
+      - release-**
+
+jobs:
+  conflicts:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```
For consistency with the rest of the codebase (with one exception).

Also, by not using `latest`, we are able to decide when to switch (when `26.04` is available, that is). Otherwise, we leave it up to GitHub, and hope for the best that it doesn't break our setup.

My 2¢.